### DISCUSSION TITLE
Address type warnings

### DISCUSSION THREAD

**Discussion by blackfalcon** [AlaskaAirlines/WC-Generator#555] [General Auro discussions]
# General Support Request

Update this code in the repo to remove the following warning that appears in the build log. 

```
/src/auro-select.js
  38:1  warning  The type 'auroSelect-ready' is undefined           jsdoc/no-undefined-types
  39:1  warning  The type 'auroSelect-valueSet' is undefined        jsdoc/no-undefined-types
  40:1  warning  The type 'auroFormElement-validated' is undefined  jsdoc/no-undefined-types
```

## Exit criteria

This issue can be closed once the code warning no longer appears in a build.

**Comment by Patrick-Daly-AA** [comment]
The big To Do on this topic is to assess and address this issue across all our components.
